app/src/main/res/layout/list_item_tool_card.xml

```diff
@@ -62,50 +63,57 @@
                 android:id="@+id/action_info"
                 android:layout_width="40dp"
                 android:layout_height="40dp"
+                android:layout_alignParentBottom="true"
                 android:layout_alignParentEnd="true"
                 android:layout_alignParentRight="true"
-                android:layout_centerVertical="true"
                 android:padding="8dp"
                 android:tint="@color/gt_blue"
                 app:srcCompat="@drawable/ic_info" />
 
-            <android.support.v7.widget.AppCompatTextViewImpl
-                android:id="@+id/language_parallel"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_centerVertical="true"
-                android:layout_toLeftOf="@id/action_info"
-                android:layout_toStartOf="@id/action_info"
-                android:textAppearance="@style/TextAppearance.GodTools2.Tools.Card.ParallelLanguage"
-                tools:text="French" />
-
             <RelativeLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_centerVertical="true"
                 android:layout_marginLeft="8dp"
                 android:layout_marginRight="8dp"
                 android:layout_toEndOf="@id/divider_download"
-                android:layout_toLeftOf="@id/language_parallel"
-                android:layout_toRightOf="@id/divider_download"
-                android:layout_toStartOf="@id/language_parallel">
+                android:layout_toRightOf="@id/divider_download">
+
+                <android.support.v7.widget.AppCompatTextViewImpl
+                    android:id="@+id/language_parallel"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignParentEnd="true"
+                    android:layout_alignParentRight="true"
+                    android:ellipsize="end"
+                    android:gravity="end"
+                    android:lines="1"
+                    android:maxWidth="60dp"
+                    android:textAppearance="@style/TextAppearance.GodTools2.Tools.Card.ParallelLanguage"
+                    tools:text="Bosnian (Bosnia &amp; Herzegovina)" />
```

Comment:
This is just for previewing in the editor right?

Reply:
Yeah, any `tools` prefixed attribute is for Android Studio only